src/graph.rs

```diff
@@ -478,24 +481,24 @@ impl PyGraph {
         self.node_indices()
     }
 
-    /// Return True if there is a node.
+    /// Check if the node exists in the graph.
     ///
-    /// :param int node: The index for the node
+    /// :param int node: The index of the node
     ///
-    /// :returns: True if there is a node false if there is no node
+    /// :returns: ``True`` if the node exists, ``False`` otherwise
     /// :rtype: bool
     #[pyo3(text_signature = "(self, node, /)")]
     pub fn has_node(&self, node: usize) -> bool {
         let index = NodeIndex::new(node);
         self.graph.contains_node(index)
     }
 
-    /// Return True if there is an edge between ``node_a`` and ``node_b``.
+    /// Check if there is a direct edge between ``node_a`` and ``node_b``.
```

Comment:
```suggestion
    /// Check if there is any undirected edge between ``node_a`` and ``node_b``.
```
I think we need to make clear that for undirected graphs if we do `graph.add_edge(node_a, node_b, w)` and do `graph.has_edge(node_b, node_a)` it will return True.